lib/bitcoin/BitcoinClient.ts

```diff
@@ -233,6 +266,13 @@ export default class BitcoinClient {
     return (inputSatoshiSum - outputSatoshiSum);
   }
 
+  /**
+   * Gets the public address for the linked wallet.
+   */
+  public getWalletAddressAsBuffer (): Buffer {

```

Comment:
Sanity: doesn't look like used? Remove if that's the case.

Reply:
Removing this as we are no longer going to be using the wallet address as the 'writer'. Instead it will be the publickeyhash (bitcoin's Hash160) which I will add in the future.